src/lib/components/LearnFilter/index.js

```diff
@@ -33,11 +36,13 @@ class LearnFilter extends BaseElement {
     }
 
     for (const child of learningPathsElement.children) {
-      child.classList.toggle('hidden', id && id !== child.id);
+      child.classList.toggle('hidden', !!id && id !== child.id);
```

Comment:
Ah, this freaking logic.

I think this change is a no-op, since `!!id` should always be the same truthiness or falsiness.

If you want to be very readable I'd wrap it in `Boolean(id)` though.